src/Umbraco.Core/Configuration/Models/ConnectionStrings.cs

```diff
@@ -26,6 +26,6 @@ private string umbracoDbDSN
         /// <summary>
         /// Gets or sets a value for the Umbraco database connection string..
         /// </summary>
-        public ConfigConnectionString UmbracoConnectionString { get; set; }
+        public ConfigConnectionString UmbracoConnectionString { get; set; } = new ConfigConnectionString(Constants.System.UmbracoConnectionName, null);
```

Comment:
If the `umbracoDbDSN` didn't exist in `appsettings.json`, this property was `null` and caused a lot of `NullReferenceException`s to be thrown. Although it would be better to do proper null-checks (as you could theoretically set it to `null` yourself), this was an easy fix for the initial issue.